DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/Enrol/EnrolDelegateSupervisor.cshtml

```diff
@@ -27,29 +27,6 @@
                             css-class="nhsuk-u-width-one-half"
                             default-option="--Select--"
                             select-list-options="@Model.SupervisorList" />
-            @if (Model.IsSelfAssessment && Model.SupervisorRoleList.Count() > 1)
```

Comment:
see other comment, I think we may still need this if more than 1 role is returned by the service